python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_enumeratefunction.py

```diff
@@ -0,0 +1,154 @@
+"""
+Plot enumeration rules
+----------------------
+"""
+
+# %%
+# In order to build up a *functional chaos* multivariate basis :math:`\{\psi_{\idx},\idx \in \NM\}`
+# by tensorization of univariate basis terms we need to enumerate the multi-indices :math:`\vect{\alpha} \in \mathbb{N}^{n_X}`.
+# In this example we are going to explore properties of these enumeration rules.
+# Refer also to :ref:`enumeration_strategy` in the theoric documentation.
+
+import openturns as ot
+import openturns.viewer as otv
+import math as m
+
+# %%
+# The simplest way to generate the multi-indices is list terms of total degree 0, then 1,2,3,...
+# This is named the linear enumeration rule; let us instanciate it in the 2-dimensional case.
+dim = 2
+enum_func = ot.LinearEnumerateFunction(dim)
+
+# %%
+# Print the 25 first multi-indices and their associated total degree.
+td_prev = 0
+print("#  | multi-index | total degree")
+print("---+-------------+-------------")
+for i in range(25):
+    multi_index = enum_func(i)
+    td = sum(multi_index)
+    if td > td_prev:
+        td_prev = td
+        print("---+-------------+-------------")
+    print(f"{i:2} |       {multi_index} |           {td}")
+
+# %%
+# Plot the multi-indices of the linear enumeration rule by stratas.
+# In the specific case of the linear enumerate function each strata contains
+# multi-indices of identical total degree that is also the index of the strata.
+def draw_stratas(enum_func):
```

Comment:
Please create a nice and clean docstring for this convenient function. This will help users to drag and drop it into user-defined scripts.